buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -526,6 +526,96 @@
         "failOnStandardError": "false"
       }
     },
+	{
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Initialize docker - Debian 9",
```

Comment:
It looks like you are doing another full clean build for Debian 9. Is that expected? I thought we were only doing the package build but not rebuilding everything. If we plan to rebuild everything why not have a separate build leg so we can do it in parallel?